plugins/mcreator-core/datalists/defaultfeatures.yaml

```diff
@@ -1,8 +1,10 @@
+- AmethystGeodes

```

Comment:
I can't find this in the official mappings, perhaps you wanted `DefaultCrystalFormations`?

Reply:
It's the Yarn's name, but I think the current name is more understandable for most users.